gym-unity/setup.py

```diff
@@ -1,15 +1,40 @@
 #!/usr/bin/env python
 
+import os
+import sys
 from setuptools import setup, find_packages
+from setuptools.command.install import install
+
+VERSION = "0.10.1"
+
+
+class VerifyVersionCommand(install):
```

Comment:
I don't have a good way to share this across the libraries :/

Reply:
:'(

Reply:
New package `ml-agents-setup-common` :troll:

Reply:
It crossed my mind.